schema/ddl/20240312160153_create_store.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS stores (
```

Comment:
[imo] storeでなく，shopでいきましょう！（APIと合わせたい）

```suggestion
CREATE TABLE IF NOT EXISTS shops (
```